mailpoet/assets/js/src/common/set_from_address_modal.tsx

```diff
@@ -81,7 +81,7 @@ type Props = {
 };
 
 function SetFromAddressModal({ onRequestClose, setAuthorizedAddress }: Props) {
-  const [address, setAddress] = useState(null);
+  const [address, setAddress] = useState('');
```

Comment:
Actually, I just tested, because I didn't like `useState<string>(null)` to much as well.
```
const [address, setAddress] = useState();
```
works just fine and imho improves readability.

Reply:
In TS it is the standard way to use generics to type hooks such as `useState<StateType>()`. Note that `useState<StateType>()` will result in `StateType | undefined` while `useState<StateType>(null)` adds `null` as well. Therefore the variant without `null` (where `undefined` is the default) is usually preferred (unless `null` is needed).

Reply:
Thanks! Addressed in 182fc8d